CHANGELOG.md

```diff
@@ -3,6 +3,24 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
+## [1.0.2] - 2023-03-30
+### :magic_wand: Added
+- Read-write splitting plugin example using Spring JDBC template. See example [here](./examples/AWSDriverExample/src/main/java/software/amazon/ReadWriteSplittingSpringJdbcTemplateExample.java)
+- Read-write splitting connection plugin benchmark and performance results ([PR #340](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/340) and [PR #316](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/316))
+- `auroraConnectionTracker` closes opened connections pointing to a dead node ([PR #298](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/298))
```

Comment:
```suggestion
- `auroraConnectionTracker` tracks all opened connections and closes all impacted connections after a failover ([PR #298](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/298))
```

Reply:
`auroraConnectionTracker` is the wrapperPlugin code, let's use the full plugin name instead to stay consistent with how we refer to `DriverMetaDataConnectionPlugin`